src/gas-hub.js

```diff
@@ -286,31 +300,31 @@ function initPageEvent() {
 
 function prepareCode() {
   return Promise.all([gas.getGasCode(), scm.getCode()])
-  .then((data) => {
-    const re = new RegExp(`\\${context.config.filetype}$`);
-    const files = $('.item > .gwt-Label').toArray().reduce((hash, e) => {
-      if (context.config.manifestEnabled && e.title === 'appsscript.json') {
-        hash['appsscript'] = 'appsscript.json';
-      }
-      const match = e.title.match(/(.*?)\.(gs|html)$/);
-      if (!match || !match[1] || !match[2]) return hash;
-      hash[match[1]] = match[0];
-      return hash;
-    }, {});
-    const code = {
-      gas: data[0].reduce((hash, elem) => {
-        if (elem && files[elem.file]) hash[files[elem.file].replace(/\.gs$/, context.config.filetype)] = elem.content;
-        return hash;
-      }, {}),
-      scm: data[1].reduce((hash, elem) => {
-        if (elem) {
-          hash[elem.file] = elem.content;
+    .then((data) => {
+      const re = new RegExp(`\\${context.config.filetype}$`);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/gas-hub.js#L304"></span>‘re’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>